terraform/production/main.tf

```diff
@@ -26,27 +26,41 @@ provider "aws" {
   }
 }
 
-module "networking" {
-  source = "../modules/networking"
-
-  prefix                                  = "core-prod"
-  vpc_cidr_block                          = "10.0.0.0/16"
-  vpc_flow_cloudwatch_log_expiration_days = 90
+locals {
+  prefix = "core-prod"
 }
 
 module "database" {
   source = "../modules/rds"
 
-  prefix                            = "core-prod"
+  prefix                            = local.prefix
   allocated_storage                 = 100
-  db_subnet_group_name              = module.networking.private_subnet_group_name
+  db_subnet_group_name              = module.networking.db_private_subnet_group_name
   instance_class                    = "db.t3.small"
   ingress_source_security_group_ids = []
   vpc_id                            = module.networking.vpc_id
 }
 
+module "networking" {
+  source = "../modules/networking"
+
+  prefix                                  = local.prefix
+  vpc_cidr_block                          = "10.0.0.0/16"
+  vpc_flow_cloudwatch_log_expiration_days = 90
+}
+
+module "redis" {
+  source = "../modules/elasticache"
+
+  prefix                  = local.prefix
+  node_type               = "cache.t2.micro"

```

Comment:
PaaS uses `micro-6.x` across all envs, which looks roughly comparable to `cache.t2.micro` in AWS so I've done for all envs here